packages/trader/src/Modules/Trading/Containers/trade-params-mobile.tsx

```diff
@@ -1,17 +1,67 @@
 import 'Sass/app/modules/trading-mobile.scss';
-
 import { Div100vhContainer, Modal, Money, Tabs, ThemedScrollbars, usePreventIOSZoom } from '@deriv/components';
-
-import AmountMobile from 'Modules/Trading/Components/Form/TradeParams/amount-mobile.jsx';
-import Barrier from 'Modules/Trading/Components/Form/TradeParams/barrier.jsx';
+import AmountMobile from 'Modules/Trading/Components/Form/TradeParams/amount-mobile';
+import Barrier from 'Modules/Trading/Components/Form/TradeParams/barrier';
 import DurationMobile from 'Modules/Trading/Components/Form/TradeParams/Duration/duration-mobile.jsx';
-import LastDigit from 'Modules/Trading/Components/Form/TradeParams/last-digit.jsx';
+import LastDigit from 'Modules/Trading/Components/Form/TradeParams/last-digit';
+import { TTextValueStrings } from '../../../Types/common-prop.type';
```

Comment:
```suggestion
import { TTextValueStrings } from 'Types/common-prop.type';
```

Reply:
this `common-prop.type` is within the trader package itself so it doesn't have aliasing i believe. Same with how its imported in useTraderStores

Reply:
this is what i have on my branch


```
const ALIASES = {
    _common: path.resolve(__dirname, '../src/_common'),
    App: path.resolve(__dirname, '../src/App'),
    Assets: path.resolve(__dirname, '../src/Assets'),
    Constants: path.resolve(__dirname, '../src/Constants'),
    Fonts: path.resolve(__dirname, '../src/public/fonts'),
    Images: path.resolve(__dirname, '../src/public/images'),
    Modules: path.resolve(__dirname, '../src/Modules'),
    Sass: path.resolve(__dirname, '../src/sass'),
    Services: path.resolve(__dirname, '../src/Services'),
    Stores: path.resolve(__dirname, '../src/Stores'),
    Translations: path.resolve(__dirname, '../src/public/translations'),
    Utils: path.resolve(__dirname, '../src/Utils'),
};
```
No aliase for Types folder
packages/trader/src/Types/common-prop.type.ts

Reply:
@henry-deriv hm, i saw it in some other PR perhaps. May I ask you to add it and change its imports?🙏🏻

Reply:
👍

Reply:
done